plugin/skills/explore-api.md

```
@@ -0,0 +1,91 @@
+Summarize the schemas by name and type. To focus on component schemas (not inline ones):
+
+```json
+{"spec": {"file": "<path>"}, "component": true}
+```
+
+## Step 4: Drill into specifics
+
+Based on what the user is interested in, drill deeper:
+
+**Specific endpoint details:**
+```json
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)